packages/astro/package.json

```diff
@@ -107,6 +107,9 @@
     "@babel/types": "^7.18.4",
     "@proload/core": "^0.3.2",
     "@proload/plugin-tsm": "^0.2.1",
+    "@types/babel__core": "^7.1.19",
+    "@types/html-escaper": "^3.0.0",
+    "@types/yargs-parser": "^21.0.0",
```

Comment:
Those three deps are not devDeps in Astro, because we ship `.d.ts` files that refer to them to users. So they need to be included when you use Astro